### ISSUE TITLE
 themes.jekyllbootstrap.com gives 404 not found

### ISSUE THREAD

**Issue by alexspeller** [plusjade/themes.jekyllbootstrap.com#1]
No worky :(

**Comment by mralexgray** [comment]
WOE IS ME.   HELP!

![Google Chrome](https://f.cloud.github.com/assets/262517/205694/11eec754-81b5-11e2-9a4a-cc0c7957f442.png)

**Comment by rmoriz** [comment]
+1

**Comment by rmoriz** [comment]
looks like themes.jekyllbootstrap.com is not pointing to the github pages ip (207.97.227.245) anymore (DNS)

**Comment by plusjade** [comment]
Yep it wasn't compiling for me when I updated it recently.   I had to
switch  to my  own  vps and only ended up finishing the main site.

I'm thinking  of  redoing the  theme integration because now it's  quite
hacky  and non intuitive.   If I update  the  integration api the idea
would  be to run a theme garden that  would  be  update- able entirely by
the community.

I belive I'm motivated to try that this weekend  =p

**Comment by yanhaijing** [comment]
404 + 1
